src/main/view/GameModelAdapter.java

```diff
@@ -0,0 +1,18 @@
+package view;
```

Comment:
No This is just so that the View doesn't have to interact with the game directly.